pkg/resources/external_function_test.go

```diff
@@ -162,6 +131,103 @@ func TestExternalFunctionDelete(t *testing.T) {
 	})
 }
 
+func TestExternalFunctionUpdateComment(t *testing.T) {
+	// r := require.New(t)
+
+	// d := externalFunction(t, "database_name|schema_name|my_test_function|varchar", map[string]interface{}{"name": "my_test_function", "arg": []interface{}{map[string]interface{}{"name": "data", "type": "varchar"}}, "return_type": "variant", "comment": "mock comment"})
+
+	WithMockDb(t, func(db *sql.DB, mock sqlmock.Sqlmock) {
+		providers := providers()
+		providers["snowflake"].ConfigureFunc = func(s *schema.ResourceData) (interface{}, error) {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>SA1019: providers["snowflake"].ConfigureFunc is deprecated: Please use ConfigureContextFunc instead. (staticcheck)